test/test_helper.rb

```diff
@@ -100,3 +101,22 @@ def auth_teardown
     end
   end
 end
+
```

Comment:
I wonder if we should leave this helper in the `ml_citation_test` file for now until it is needed elsewhere. It seems unlikely that we'd actually need this helper anywhere else so it feels a bit more complex to have to look here rather than just having it at the top of the file being tested? This is not a strong feeling so if you feel it is better here that works for me.

Reply:
Oh, I see. You use it it both ml_citation_test and term_test already. I'm leaving my comment so you can see what I was thinking... however, I now think this is the correct location for this helper.